components/ceph/config.pan

```diff
@@ -16,7 +16,7 @@
 
 unique template components/ceph/config;
 
-variable CEPH_SCHEMA_VERSION ?= 'v1';
+variable CEPH_SCHEMA_VERSION ?= 'v2';
```

Comment:
Not a blocker but I think needs to get fixed here too or I think it will get reverted at next release: https://github.com/quattor/configuration-modules-core/blob/400aba5c9c5c344fc8b88f0a54209d038fbaedaa/ncm-ceph/src/main/pan/components/ceph/config.pan

Reply:
https://github.com/quattor/configuration-modules-core/pull/1506